src/keylessBackup/SignInWithEmail.tsx

```diff
@@ -234,8 +234,8 @@ function SignInWithEmail({ route }: Props) {
             onPress={() => onPressSignIn('apple')}
             text={t('signInWithEmail.apple')}
             size={BtnSizes.FULL}
-            type={BtnTypes.PRIMARY}
-            icon={<AppleIcon color={Colors.white} />}
+            type={BtnTypes.SECONDARY}
+            icon={<AppleIcon color={Colors.black} />}
```

Comment:
just checking that we want to make these changes for onboarding too? kind of seems like we just restyled the onboarding flow, so i'm surprised we're changing it again 😅

Reply:
According to [the designs](https://www.figma.com/design/kXwIdFqrwvFOaI63rNRCAb/Mobile-Stack-Run-time?node-id=2521-14302&t=IuanyqfGijz4RUl6-4), these buttons are changed in onboarding as well, so I imagine that this change ought to exist in both places.